app/controllers/logs_controller.rb

```diff
@@ -25,7 +25,7 @@ def valid_search_params
   def get_auth_requests
     UseCases::Administrator::GetAuthRequests.new(
       authentication_logs_gateway: Gateways::Sessions.new(
-        ips: current_organisation.ips.map(&:address)
+        ip_filter: current_user.super_admin? ? nil : current_organisation.ips.map(&:address)
```

Comment:
```suggestion
        ip_filter: current_organisation.ips.map(&:address) if !current_user.super_admin?
```

Reply:
Prefer the my version and its explicit as were passing this in 3 places so it is nicer to see what you're actually passing.